### PR TITLE
Create PPP first draft- Syed Omar Zoran

### PR DESCRIPTION
**Pull request by zoranabc201**

### CODE REVIEW

File: docs/team/syedomarzoran.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+## Property Rental Manager - Overview
+Property Rental Manager (PRM) is a desktop application that helps property agents manage, filter and 
+monitor single owner rental units for appropriate tenants.
+
+### Summary of Contributions -
+#### Code contributed - 
+[RepoSense Link](https://nus-cs2113-ay2223s1.github.io/tp-dashboard/?search=zoranabc201&breakdown=true)
+#### Enhancements implemented -
+* Added all the list features for the app -
```
Comment (ngdeqi) [inline docs/team/syedomarzoran.md:11]: You have also contributed the quit feature

File: docs/team/syedomarzoran.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+[RepoSense Link](https://nus-cs2113-ay2223s1.github.io/tp-dashboard/?search=zoranabc201&breakdown=true)
+#### Enhancements implemented -
+* Added all the list features for the app -  
+PRM is an app with extensive options for listing the content stored in it - 
+  * Clients - Clients can be listed in various ways. All information present can be listed. A condensed version
+    (called -short) lists the most pertinent information. Individual attributes of clients can also be listed.
+  * Properties - Similar to clients. 
+  * Pairs - Allows listing information about all pairs present in the app. Pairs can be listed either with full 
+  information about their clients and properties, or with condensed information only. This was implemented as a
+  response to a PE-D suggestion and had to be done quickly. Listed items here are not numbered as pairings are not numbered
```
Comment (wilsonngja) [inline docs/team/syedomarzoran.md:18]: I think more elaboration on the complexity of the implementation can be added if needed. I believe you can also add in the tag function if you like to.

File: docs/team/syedomarzoran.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  internally, and there was no time to change that before V2.1. Therefore, all members of the HashMap which stores
+  the pairings are displayed, in no particular order.
+* Added exceptions and messages related to the list function
+#### Contributions to the UG
+* Added information for the list section of the UG -  
+Described the enhancements I added in the UG.
```
Comment (wilsonngja) [inline docs/team/syedomarzoran.md:24]: Perhaps can consider being more explicit in your explanation?

**Review by wilsonngja — COMMENTED** [review]
I agree with what you have done thus far and the things you've stated in the PPP.
You might want to be more clear in your explanation in your UG and DG section.
Overall, good job.

**Review by OVReader — APPROVED** [review]
Overall LGTM for PPP first draft as most of the required details are mentioned. Do consider the suggestions given by @wilsonngja and @ngdeqi for the final version.

**Pull request merged**